### PR TITLE
adding edit/destroy a theme

### PR DESCRIPTION
**Pull request by marjo1986**
i added edit and destroy at theme_controller. Can you review the methods and merge if is good?

### CODE REVIEW

File: app/controllers/themes_controller.rb [hunk 1]
```diff
@@ -21,6 +21,18 @@ def create
     end
   end
 
+  def edit
+    @theme = Theme.find(params[:id])
+    @theme.update(params[:theme])
+    redirect_to theme_path(@theme)
+  end
```
Comment (Rom1023) [inline app/controllers/themes_controller.rb:32]: You should only display the form to edit the theme by doing the following code :
def edit
@theme = Theme.find(params[:id])
end

File: app/controllers/themes_controller.rb [hunk 2]
```diff
@@ -21,6 +21,18 @@ def create
     end
   end
 
+  def edit
```
Comment (vivien-z) [inline app/controllers/themes_controller.rb:24]: 1. edit just need to pass the theme to view 2.missing the update step. that is where you need to add the update

File: app/controllers/themes_controller.rb (hunk 1 above)
Comment (Rom1023) [reply app/controllers/themes_controller.rb:32]: Then you create a second action called update

File: app/controllers/themes_controller.rb [hunk 3]
```diff
@@ -21,6 +21,18 @@ def create
...
+    @theme = Theme.find(params[:id])
+    @theme.update(params[:theme])
+    redirect_to theme_path(@theme)
+  end
+
+  def destroy
+    @theme = Theme.find(params[:id])
+    @theme.destroy
+    redirect_to theme_path(@theme)
```
Comment (Rom1023) [inline app/controllers/themes_controller.rb]: Once you destroyed the theme you should redirect to all the themes

**Comment by marjo1986** [comment]
correction done review again

File: app/controllers/themes_controller.rb [hunk 4]
```diff
@@ -23,14 +23,18 @@ def create
...
+    @theme = Theme.find(params[:id])
+    @theme.update(theme_params)
     redirect_to theme_path(@theme)
   end
 
   def destroy
     @theme = Theme.find(params[:id])
     @theme.destroy
-    redirect_to theme_path(@theme)
+    redirect_to themes_path
```
Comment (Rom1023) [inline app/controllers/themes_controller.rb:37]: Looks good

**Review by Rom1023 — COMMENTED** [review]
Looks good to me

**Pull request merged**